recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
     - 0001-remove-wil-from-exported-config.patch
 
 build:
-  number: 1
+  number: 2
```

Comment:
Same comment as in https://github.com/conda-forge/azure-core-cpp-feedstock/pull/16#discussion_r1503219345. The rerendering didn't change any of the pins. It is only removing the extra build variants that we no longer need

Reply:
Reverted.